howto/update/upgrade-anbox.md

```diff
@@ -37,39 +62,87 @@ The deployed Juju charms need to be upgraded next.
 
 [/note]
 
-Run the following commands in the exact same order as listed here but skip those you don't use in your deployment:
+For any of the charm upgrades you can watch the upgrade status by running
+
+    juju status
+
+Only continue with the next step once the current step has completed successfully and all units in the output are marked as **active**.
+
+[note type="information" status="Note"]
+If you don't run Anbox Cloud in a high availity configuration, upgrading the charms will cause a short down time of individual service components during the process.
+[/note]
+
+### Upgrade infrastructure components
+
+As first step we will update all infrastucture components. This includes deployed internal certificate authorities and etcd.
+
+First we update easyrsa:
+
+    juju refresh internal-ca --revision=26
+    juju refresh etcd-ca --revision=26
+
+Once the upgrade has completed, you can upgrade the etcd service next:
+
+    juju refresh etcd --revision=724
```

Comment:
Do we really need to specify the revision number? Isn't there a shortcut or alias?
Otherwise, this means we have to make sure we update the revision for every release.

Reply:
We pin particular versions of the charm with each Anbox Cloud release as that is what we have tested with and qualified. So yes, we have to specify the exact revision here.